sdk-base/src/main/java/com/mapbox/maps/MapboxLogger.kt

```diff
@@ -0,0 +1,27 @@
+@file:JvmName("MapboxLogger")
+
+package com.mapbox.maps
+
+import com.mapbox.common.Logger
+
+/**
+ * Log Android error entry in the following format: E/Mapbox: [[SDK_IDENTIFIER]\ [tag]] [message]
+ */
+fun logE(tag: String, message: String) = Logger.e("$SDK_IDENTIFIER\\$tag", message)
```

Comment:
So previously it was plain `tag`, now tag is `maps-android/tag`, where is `Mapbox` originating from? Is it inserted in common?

Reply:
you're totally correct